File: QuickStart/TrackStatusControls/ShapedForm/FirstLook/MyShapedForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Telerik.WinControls;

namespace Telerik.Examples.WinControls.UIElements.ShapedForm.FirstLook
{
	public partial class MyShapedForm : Telerik.WinControls.UI.ShapedForm
	{
		public MyShapedForm(): this(null)
		{
		}

		public MyShapedForm(ElementShape shape)
		{
			InitializeComponent();
			if (shape != null)
				this.Shape = shape;
		}

		private void radButton1_Click(object sender, EventArgs e)
		{
			this.Close();
		}

		protected override int GetHitTest(Point point)
		{
			int result = base.GetHitTest(point);
			if (result == 1)
				result = 2;
			return result;
		}
	}
}

File: QuickStart/TrackStatusControls/ShapedForm/FirstLook/MyShapedForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports Telerik.WinControls

Namespace Telerik.Examples.WinControls.UIElements.ShapedForm.FirstLook
	Partial Public Class MyShapedForm
		Inherits Telerik.WinControls.UI.ShapedForm
		Public Sub New()
			Me.New(Nothing)
		End Sub

		Public Sub New(ByVal shape As ElementShape)
			InitializeComponent()
			If shape IsNot Nothing Then
				Me.Shape = shape
			End If
		End Sub

		Private Sub radButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles radButton1.Click
			Me.Close()
		End Sub

		Protected Overrides Function GetHitTest(ByVal point As Point) As Integer
			Dim result As Integer = MyBase.GetHitTest(point)
			If result = 1 Then
				result = 2
			End If
			Return result
		End Function
	End Class
End Namespace